3/3scale_toolbox/build.sh

```diff
@@ -0,0 +1,50 @@
+# ----------------------------------------------------------------------------
+# Package       : 3scale_toolbox 
+# Version       : v0.18.1
```

Comment:
Can you kindly point to this Tag/Branch 3scale-2.10.0-GA?
https://github.com/3scale/3scale_toolbox/archive/refs/tags/3scale-2.10.0-GA.tar.gz